domain/domain.go

```diff
@@ -405,16 +407,16 @@ func (do *Domain) loadSchemaInLoop(lease time.Duration) {
 		case <-syncer.Done():
 			// The schema syncer stops, we need stop the schema validator to synchronize the schema version.
 			log.Info("[ddl] reload schema in loop, schema syncer need restart")
-			do.SchemaValidator.Stop()
 			err := do.mustRestartSyncer()
 			if err != nil {
 				log.Errorf("[ddl] reload schema in loop, schema syncer restart err %v", errors.ErrorStack(err))
 				break
 			}
-			do.SchemaValidator.Restart()
```

Comment:
Need to add metrics here?

Reply:
we already has metrics.NewSessionHistogram and metrics.DeploySyncerHistogram, I think there is nothing need to add here.

Reply:
Got it.